### PR TITLE
Finish mobile version

### PR DESCRIPTION
**Pull request by adeola003**
In order to finish the portfolio mobile version project, here are the steps I followed :
1. I create a new branch from the previous merge name finish_mobile_version;
2. I added the new sections to the HTML file following the chosen template in the previous project;
3. I added the style according to the Figma template;
4. I checked the installation of the linters and fixed the detected errors.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -23,8 +23,8 @@
                 <span class="line2"></span>
                 <span class="line3"></span>
             </div>
-            <a href="#news">Portfolio</a>
```
Comment (joel-kalema) [inline index.html:26]: - [ ] As I can see you did great work, kindly make sure to stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

    - As you can see the Figma design, your margins are note exactly the some 
    - Kindly, make sure your `header` look like the Figma design.  
   
> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |mobile 2 0 | |  | |

- [ ] Kindly, make sure you remove the green background 👍 
- [ ] Kindly, make sure your images are running 👍 
- [ ]  Kindly, make sure you fixed margins and look like the Figma design

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | | | |  | |



- [ ] Kindly, make sure you fixed the language images and fix the flex-direction for languages and Frameworks 👍 
- [ ] Kindly, make sure you fixed the margins 👍 


> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | | | |  | |

**Review by joel-kalema — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @adeola003 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @joel-kalema  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by joel-kalema — APPROVED** [review]
Hi @adeola003 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters are passing 👍 
The project is running successfully 👍 
PR is well descriptive 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@joel-kalema** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by adeola003** [comment]
Thanks you very much @joel-kalema

**Pull request merged**